movie_rec/recommender_server.py

```diff
@@ -0,0 +1,66 @@
+import json
+from optparse import OptionParser
+import psycopg2
+
+class MovieRecommender(object):
```

Comment:
i don't think there is strict convention in python. just got rid of it for the server one.